beluga/package.xml

```diff
@@ -15,6 +15,7 @@
   <buildtool_depend>cmake</buildtool_depend>
 
   <depend>eigen</depend>
+  <depend>libhdf5-dev</depend>
```

Comment:
Is this not available in Focal or something? :smiling_face_with_tear:

Reply:
It is! It seems like the old version doesn't export the `hdf5::*` variables, but we should be good to go using the other one I just switched to.
https://cmake.org/cmake/help/latest/module/FindHDF5.html

Reply:
It is! https://packages.ubuntu.com/search?keywords=libhdf5-dev

I used another variable to link against the libraries based on:
https://cmake.org/cmake/help/latest/module/FindHDF5.html
That seems to be part of all the distributed versions.

Reply:
Sweet!